### ISSUE TITLE
Flatpak build

### ISSUE THREAD

**Issue by LeandroStanger** [ctt-gob-es/clienteafirma#122]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing Cliente afirma on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

**Comment by EchedeyLR** [comment]
I would agree with that too. In this way the support would be for a wide variety of distributions.
